File: reni-frontend/src/main/reni-frontend/src/app/modules/intranet/dto/request/estructura-modificar.request.ts
import { EstructuraModificarParametroRequest } from "./estructura-modificar-parametro.request";
import { EstructuraRegistrarParametroRequest } from "./estructura-registrar-parametro.request";

export class EstructuraModificarRequest {
    idEstructura: number;
    nombre: string;
    descripcion: string;
    idtTipoUsuario: string;
    idServicio: number;
    idEstructuraRepl: number;
    listaParametro: EstructuraRegistrarParametroRequest[];
    listaParametroMod: EstructuraModificarParametroRequest[];
    listaParametroElim: number[];
    idUsuarioModif: number;
}

File: reni-backend/src/main/java/pe/gob/mimp/reni/dto/request/EstructuraModificarRequest.java
package pe.gob.mimp.reni.dto.request;

import java.io.Serializable;
import java.util.List;

import lombok.Getter;
import lombok.ToString;
import lombok.Setter;

@Getter
@Setter
@ToString
public class EstructuraModificarRequest implements Serializable {
	private static final long serialVersionUID = 1L;

	private Long idEstructura;
	private String nombre;
	private String descripcion;
	private String idtTipoUsuario;
	private Long idServicio;
	private Long idEstructuraRepl;
	private List<EstructuraRegistrarParametroRequest> listaParametro;
	private List<EstructuraModificarParametroRequest> listaParametroMod;
	private List<Long> listaParametroElim;
	private Long idUsuarioModif;
}
